apps/language_server/lib/language_server/experimental/source_file/conversions.ex

```diff
@@ -102,6 +103,19 @@ defmodule ElixirLS.LanguageServer.Experimental.SourceFile.Conversions do
     {:ok, range}
   end
 
+  def to_lsp(
+        %ElixirSense.Location{line: line, column: column} = elixir_sense_location,
```

Comment:
nit: don't pattern match out struct fields in function heads unless you are doing something with the values, it makes the functions a lot harder to read and doesn't provide an advantage over using the dot syntax.

Instead, favor `location.line` and `location.column` in the function body.